respy/python/simulate/simulate_python.py

```diff
@@ -63,133 +74,125 @@ def pyth_simulate(
             optim_paras["shocks_cholesky"],
         )
 
-    # We also need to sample the set of initial conditions.
+    # Get initial values of SCHOOLING, lagged choices and types for simulated agents.
     initial_education = get_random_edu_start(
         edu_spec, num_agents_sim, is_debug
     )
+    initial_choice_lagged = get_random_choice_lagged_start(
+        edu_spec, num_agents_sim, initial_education, is_debug
+    )
     initial_types = get_random_types(
         state_space.num_types,
         optim_paras,
         num_agents_sim,
         initial_education,
         is_debug,
     )
-    initial_choice_lagged = get_random_choice_lagged_start(
-        edu_spec, num_agents_sim, initial_education, is_debug
-    )
+
+    # Create a matrix of initial states of simulated agents. OCCUPATION A and OCCUPATION
+    # B are set to zero.
+    current_states = np.c_[
```

Comment:
could we use np.column_stack() instead of np.c_[]? I would find that more readable. You can decide.

Reply:
I agree.